ui/partials/main/thPinboardPanel.html

```diff
@@ -66,23 +66,30 @@
      title="{{!hasPinnedJobs() ? 'No jobs pinned' : ''}}">
   <div class="btn-group">
     <button class="btn btn-default btn-xs save-btn"
-          title="{{ !hasPinnedJobs() ? 'No job selected' : 'Save all classification data' }}"
-          ng-click="save()"
-          ng-disabled="!canSaveClassifications()">save
+            title="{{ !user.loggedin && !canSaveClassifications() ?
+                      'Not logged in / default classification data / no job pinned' : 'Save all classification data' }}"
```

Comment:
Or maybe define a helper function in pinboard.js to generate this string for you. that would be legit :)